Maybe/Maybe.cs

```diff
@@ -37,7 +37,7 @@ public T Value
             {
                 if (!HasValue)
                 {
-                    throw new Exception("No Value is present");
+                    throw new Exception($"Maybe<{typeof(T).Name}>: No Value is present");
```

Comment:
Looks like we're missing a test for this case. Please add one.

Reply:
The tests already check the actual error in the message ("No Value is present"). You think it's necessary to check every instance of message type?
I believe that if we are checking that an exception has been thrown and the message contains "No Value is present" it's safe to assume it was thrown here, we don't need to test the interpolation

Reply:
Oh I rechecked it, this is only happening to the "Sequence" tests, I'll add one for this then